pkg/collector/corechecks/cluster/orchestrator/discovery/collector_discovery.go

```diff
@@ -110,6 +110,11 @@ func (p *APIServerDiscoveryProvider) walkAPIResources(inventory *inventory.Colle
 				continue
 			}
 
+			// Ignore unstable collectors.
+			if !collector.Metadata().IsStable {
```

Comment:
Are there other things that should be checked for stability or is this more of a one off?